app/javascript/packs/reducers/restaurants/actions.js

```diff
@@ -0,0 +1,47 @@
+import { getFiltersHash } from './selectors';
+import { searchRestaurants as searchRestaurantsApi } from '../../api/restaurants';
+import { hash } from '../../modules/hash';
+
+const ACTIONS_NAME = {
+    SEARCH_RESTAURANTS_STARTED: 'restaurants-search-started',
+    SEARCH_RESTAURANTS_DONE: 'restaurants-search-done',
+    SEARCH_RESTAURANTS_FAILED: 'restaurants-search-failed',
+};
+
+const searchRestaurantsStarted = (filtersHash, isInBackground) => ({
+    type: ACTIONS_NAME.SEARCH_RESTAURANTS_STARTED,
+    filtersHash,
+    isInBackground,
```

Comment:
I personally recommend adhering to the [FSA spec](https://github.com/redux-utilities/flux-standard-action) (which `redux-actions` does for you).

Reply:
I didn't know there is a standard, will follow moving forward.